os_migrate/tests/unit/test_server.py

```diff
@@ -259,6 +259,86 @@ def test_block_device_mapping_boot_disk_nocopy(self):
         ])
         self.assertTrue('image_id' not in sdk_params)
 
+    # TODO: add testcase for no data copy, no boot copy, load from optional image
```

Comment:
I've added a test case below to match the condition in the CI when boot-volume, additional volumes is empty, boot-disk-copy and data-copy are false like shown here: 

```
 _migration_params:
        additional_volumes:
        - uuid: null
        boot_disk_copy: false
        boot_volume:
          uuid: null
        boot_volume_params:
          availability_zone: null
          description: null
          name: null
          volume_type: null
        data_copy: false
        floating_ip_mode: auto
        port_creation_mode: neutron
```